src/org/usfirst/frc/team2335/robot/triggers/Axis.java

```diff
@@ -0,0 +1,48 @@
+package org.usfirst.frc.team2335.robot.triggers;
+
+import org.usfirst.frc.team2335.robot.Robot;
+
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj.buttons.Trigger;
+
+public class Axis extends Trigger
+{
+	//Constants
+	public static final double DEADZONE = 0.2;
+	
+	//Axes
+	public static final int MOVE = 1, ROTATE = 0;
+	
+	//Motors
+	public static final int LEFTDRIVE = 1, RIGHTDRIVE = 0, ARM = 8;
+	
+	//Buttons
+	public static final int ARMDOWN = 4, ARMUP = 6;
+
```

Comment:
There's no reason to use these constants. They're never referenced in any other file. Correct me if I'm wrong though.